dependabot-automerge/action.yml

```diff
@@ -34,19 +34,24 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - if: github.event_name == 'pull_request' && github.actor == 'dependabot[bot]'
+    - if: >
+        (github.event_name == 'pull_request' || github.event_name == 'pull_request_target')
```

Comment:
We should support both `pull_request` and `pull_request_target` even though we run our own just with `pull_request`.